MainDemo.Wpf/Buttons.xaml

```diff
@@ -45,335 +46,596 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
-        </Grid.RowDefinitions>        
-        <TextBlock
+        </Grid.RowDefinitions>
+        <DockPanel LastChildFill="False">
+            <TextBlock
             Style="{StaticResource MaterialDesignHeadline5TextBlock}"
             Text="Buttons"/>
+            <ToggleButton x:Name="ButtonsEnabled"
+                    Style="{StaticResource MaterialDesignSwitchToggleButton}"
+                    ToolTip="MaterialDesignSwitchLightToggleButton"
+                    IsChecked="True" DockPanel.Dock="Right"
+                    Margin="8 2"/>
+            <TextBlock VerticalAlignment="Center" Style="{StaticResource MaterialDesignBody1TextBlock}" Margin="8 0 0 0"
+                       DockPanel.Dock="Right">Enabled</TextBlock>
+
+        </DockPanel>
+
         
         <StackPanel Grid.Row="1">
-            <StackPanel
+            <StackPanel.Resources>
+                <system:Double x:Key="IconSize">24</system:Double>
+                <system:Double x:Key="ButtonWidth">90</system:Double>
+                <ObjectDataProvider x:Key="ColorZoneModes" MethodName="GetValues" ObjectType="{x:Type sys:Enum}">
+                    <ObjectDataProvider.MethodParameters>
+                        <x:Type TypeName="materialDesign:ColorZoneMode" />
+                    </ObjectDataProvider.MethodParameters>
+                </ObjectDataProvider>
+            </StackPanel.Resources>
+
+            <materialDesign:ColorZone Margin="0 24 0 0" Mode="{Binding SelectedItem, ElementName=ColorZoneMode}">
+                <StackPanel>
+                    <DockPanel>
+                        <ComboBox DockPanel.Dock="Right" Name="ColorZoneMode" HorizontalAlignment="Center" VerticalAlignment="Center" MinWidth="120" ItemsSource="{Binding Source={StaticResource ColorZoneModes}}" SelectedIndex="0" />
```

Comment:
I am not sure how much value this adds since in most cases you just need to test dark and light theme. This is already an option from the three dots menu in the top right.

Reply:
I don't have a strong opinion on leaving it in, it was more to show how the paper variant responds to being on the colored backgrounds, which granted, is probably not a common use case really